src/test/shared/credentials/userCredentialsUtils.test.ts

```diff
@@ -93,6 +93,22 @@ describe('UserCredentialsUtils', function () {
             assert(foundFiles)
             assert.strictEqual(foundFiles.length, 0)
         })
+
+        it('returns empty result if files dont contain credentials', async function () {
+            const credentialsFilename = path.join(tempFolder, 'credentials-both-exist-but-no-credentials-test')
+            const configFilename = path.join(tempFolder, 'config-both-exist-but-no-credentials-test')
```

Comment:
These files don't exist, which is worth testing. But what about the case where the files exist but

1. they contain nonsense
2. they are empty
3. they contain valid profiles but the profiles are not listable (e.g. `[sso-profile foo]`?